### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by iLinuxx**
https://ilinuxx.github.io/layout_landing-page/

### CODE REVIEW

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Please provide DEMO_LINK in the description
image

Make sure you have deployed your solution
image

If you need any kind of help - feel free to ask mentors in the chat

**Comment by iLinuxx** [comment]
https://ilinuxx.github.io/layout_landing-page/

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good start!

1. Hide scroll when menu is open
image

Please, re-request review when the entire task will be done (check yourself using the CHECKLIST when finished)

If you have any questions feel free to ask in the chat

**Comment by iLinuxx** [comment]
> Good start!
> 
> 1. Hide scroll when menu is open
>    image
> 
> Please, re-request review when the entire task will be done (check yourself using the CHECKLIST when finished)
> 
> If you have any questions feel free to ask in the chat

in checklist items: 
14. Placeholders should give examples of what should be added, to make the expected format clear
15. Page shouldn't be reloaded on form submit, but the fields should be cleared. Use:
    ```html
    <form onsubmit="this.reset(); return false;">
    ```
16. Make `footer` sticky at the bottom with `z-index: -1` to get fixed background effect.

but in the BEM theme we didn't do these elements, should I add it?

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Add phone number here to be able to call
image
Add transition when this menu appears at page
image
Add a favicon to page
image

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://iLinuxx.github.io/layout_landing-page/)
```
Comment (volodymyr-soltys97) [inline README.md:14]: Add this link to PR description

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
You didn't push anything since last review. Please make re-request when you fully complete your landing page.

**Pull request closed without merging**